src/js/feat.js

```diff
@@ -1273,7 +1309,7 @@ const editability = {
 
             function endEditing() {
                 component.applyUserAction(Level.A, () => {
-                    component.propagate({ 'text': textareaEl.value });
+                    component.propagate({ 'text': textareaEl.value, 'textColor': textColorEl.value, 'color': backColorEl.value });
```

Comment:
It looks like another functionality is pulled together.  I'll remove these from this PR.